docs/components/radio_buttons.md

```diff
@@ -21,11 +21,15 @@ Use radio buttons to:
 
 ## Contents
 
-- [Using radio buttons](#using-radio-buttons)
-- [Import radio buttons](#import-radio-buttons)
-- [Making radio buttons accessible](#making-radio-buttons-accessible)
-- [Radio buttons example](#radio-buttons-example)
-- [Theming](#theming)
+- [Selection controls: radio buttons](#selection-controls-radio-buttons)
+  - [Contents](#contents)
+  - [Using radio buttons](#using-radio-buttons)
+    - [Import radio buttons](#import-radio-buttons)
+    - [Making radio buttons accessible](#making-radio-buttons-accessible)
+  - [Radio buttons](#radio-buttons)
+    - [Radio buttons example](#radio-buttons-example)
+    - [Radio button states](#radio-button-states)
+  - [Theming](#theming)
```

Comment:
Are you sure we want to change this? I don't think we want the `contents` item

Reply:
Done